draft-hodges-webauthn-registries.xml

```diff
@@ -148,19 +148,13 @@ https://github.com/w3c/webauthn/pulls?q=is%3Apr+label%3Aspec%3Awebauthn-registri
         this document and, if defined, the processes established
         by the expert(s); typically, this will mean referring them to the registry Web page.
       </t>
-      <t>
-        Note that the expert(s) are allowed (as per
-        <xref target="sctn-registering-attstn-format-idents"/> and
-	<xref target="sctn-registering-extension-idents"/>) to define
-        additional fields to be collected in the registry.
-      </t>
 
       <section title="WebAuthn Attestation Statement Format Identifier Registry"
         anchor="sctn-attstn-format-registry">
         <t>
           WebAuthn attestation statement format identifiers are strings whose semantic, syntactic,
           and string-matching criteria are specified in <xref target="WebAuthn"/>
-          <eref target="https://www.w3.org/TR/webauthn/#sctn-attstn-fmt-ids">
+          <eref target="https://www.w3.org/TR/2019/REC-webauthn-1-20190304/#sctn-attstn-fmt-ids">
```

Comment:
hm....yeah, ok, linking to specific version is ok given what this here doc is doing (reg'g things in IANA)